docker-compose/merging/cgmes-assembling-job/cgmes-assembling-job-config.yml

```diff
@@ -6,14 +6,13 @@ acquisition-server:
   label: opde
   dependencies-strict-mode: false
 
-cassandra:
-  contact-points: <TO_COMPLETE>
-  port: 9042
-  datacenter: datacenter1
-  keyspace-name: cgmes_assembling
-
 case-server:
   url: http://case-server/
 
 cgmes-boundary-server:
     url: http://cgmes-boundary-server/
+
+database:
+  url: jdbc:postgresql://localhost:5432/cgmes_assembling
```

Comment:
`172.17.0.1` instead of `localhost` to be the same as in docker-compose/merging/case-import-job/case-import-job-config.yml